iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/common/import/oozie_app/workflow.xml

```diff
@@ -710,16 +718,26 @@
         <java>
 			<prepare>
```

Comment:
Fixed that. Although fixing this is least favorite task of mine... I hope it won't make much pain when merging changes coming from different branches because indention changes will be picked up.